builtin/providers/openstack/resource_openstack_lb_loadbalancer_v2.go

```diff
@@ -75,7 +75,7 @@ func resourceLoadBalancerV2() *schema.Resource {
 				ForceNew: true,
 			},
 
-			"provider": &schema.Schema{
+			"loadbalancer_provider": &schema.Schema{
```

Comment:
@stack72 I can definitely do a deprecation thing here.

Just curious, though: would `provider` have worked during any previous release? Was it only recent that the error reported in #12237 would happen? If so, that would definitely help my sanity :)